### PR TITLE
(PC-12862) feat(accessibility): Use hidden checkbox with filter switch

### PR DESCRIPTION
**Pull request by aliraiki**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12862

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         |                 |                  |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/ui/web/inputs/HiddenCheckbox.web.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const HiddenCheckbox = ({
+  checked,
+  accessibilityLabel,
+}: {
+  checked: boolean
+  accessibilityLabel: string
+}) => <input type="checkbox" checked={checked} aria-label={accessibilityLabel} hidden readOnly />
```
Comment (kopax-polyconseil) [inline src/ui/web/inputs/HiddenCheckbox.web.tsx:9]: Du coup, on a besoin de `id` pour le `<label>`

Comment (aliraiki) [reply src/ui/web/inputs/HiddenCheckbox.web.tsx:9]: Mais on ne met pas de `label` si ? Juste un `aria-label`

Comment (kopax-polyconseil) [reply src/ui/web/inputs/HiddenCheckbox.web.tsx:9]: Comment est-ce qu'on va cliquer sur le label d'un switch pour activer la checkbox et le switch ?

Comment (aliraiki) [reply src/ui/web/inputs/HiddenCheckbox.web.tsx:9]: Ah je savais pas que ca rentrait dans le scope, j'ai ajouté ça dans la nouvelle PR https://github.com/pass-culture/pass-culture-app-native/pull/2593

**Pull request merged**